CONTRIBUTING.md

```diff
@@ -108,46 +90,59 @@ This will download the entire freeCodeCamp repo to your projects directory.
 1. Change directory to the new freeCodeCamp directory (`cd freeCodeCamp`)
 2. Add a remote to the official freeCodeCamp repo:
 
-```shell
-$ git remote add upstream https://github.com/freeCodeCamp/freeCodeCamp.git
-```
+    ```shell
+    git remote add upstream https://github.com/freeCodeCamp/freeCodeCamp.git
+    ```
+
+3. Confirm the available remotes:
+
+    ```shell
+    git remote -v
 
-Congratulations, you now have a local copy of the freeCodeCamp repo!
+    ## Result
+    origin git@github.com:yourUsername/curriculum.git (fetch)
+    origin git@github.com:yourUsername/curriculum.git (push)
+    upstream https://github.com/freeCodeCamp/freeCodeCamp.git (fetch)
+    upstream https://github.com/freeCodeCamp/freeCodeCamp.git (push)
+    ```
+
+_Congratulations, you now have a local copy of the freeCodeCamp repo!_
 
 #### Maintaining Your Fork
 
 Now that you have a copy of your fork, there is work you will need to do to keep it current.
 
-##### Rebasing from Upstream
+### Rebasing from Upstream
 
 Do this prior to every time you create a branch for a PR:
 
 1. Make sure you are on the `staging` branch
 
-```shell
-$ git status
-On branch staging
-Your branch is up-to-date with 'origin/staging'.
-```
-If your aren't on `staging`, resolve outstanding files / commits and checkout the `staging` branch
+    ```shell
+    $ git status
```

Comment:
`git status`

With a separate code block for the expected output.

Reply:
Totally agree with you. This is some formatting WIP. I am going to remove all of this outdated git steps. These can be made lot simpler.